### ISSUE TITLE
Fedora 32 VHD test fails with image_info errors

### ISSUE THREAD

**Issue by major** [osbuild/osbuild-composer#839]
Fedora 32's VHD test often fails with `image_info` errors like these:

Full log: [fedora_32-x86_64-vhd-boot.json.log](https://github.com/osbuild/osbuild-composer/files/4898359/fedora_32-x86_64-vhd-boot.json.log)

Excerpt:
```diff
Diff:
--- Expected
+++ Actual
@@ -463,4 +463,5 @@
    (string) (len=10) "rpm-verify": (map[string]interface {}) (len=2) {
-  (string) (len=7) "changed": (map[string]interface {}) (len=9) {
+  (string) (len=7) "changed": (map[string]interface {}) (len=10) {
    (string) (len=38) "/boot/System.map-5.6.6-300.fc32.x86_64": (string) (len=9) ".M.......",
+   (string) (len=11) "/boot/grub2": (string) (len=9) ".M.......",
    (string) (len=41) "/boot/initramfs-5.6.6-300.fc32.x86_64.img": (string) (len=9) ".M.......",
@@ -474,3 +475,783 @@
    },
-  (string) (len=7) "missing": ([]interface {}) {
+  (string) (len=7) "missing": ([]interface {}) (len=780) {
+   (string) (len=20) "/boot/efi/EFI/fedora",
+   (string) (len=26) "/etc/NetworkManager/conf.d",
+   (string) (len=42) "/etc/NetworkManager/dispatcher.d/no-wait.d",
+   (string) (len=43) "/etc/NetworkManager/dispatcher.d/pre-down.d",
+   (string) (len=41) "/etc/NetworkManager/dispatcher.d/pre-up.d",
+   (string) (len=36) "/etc/NetworkManager/dnsmasq-shared.d",
+   (string) (len=29) "/etc/NetworkManager/dnsmasq.d",
+   (string) (len=38) "/etc/NetworkManager/system-connections",
...
```

The image itself boots just fine when it runs with the integration (bash-based) tests that use libvirt.

**Comment by ondrejbudai** [comment]
#827 is now merged, let's see if it helps with this issue.

**Comment by major** [comment]
🤞🏻

**Comment by ondrejbudai** [comment]
Nope, it still happens:

https://osbuildci.cloud.paas.psi.redhat.com/blue/organizations/jenkins/osbuild%2Fosbuild-composer/detail/PR-857/2/pipeline/30/

**Comment by ondrejbudai** [comment]
I was able to capture the failed image:

https://drive.google.com/file/d/1FKTYErFanEm489MFAxDQEWHvuyMeipH9/view?usp=sharing

**Comment by teg** [comment]
> I was able to capture the failed image:

Nice! Is the failure actually in the image, or in image-info?

**Comment by msehnout** [comment]
> > I was able to capture the failed image:
> 
> Nice! Is the failure actually in the image, or in image-info?

In the image. All the files are indeed missing. For example:
```
[redhat@ibm-p8-kvm-03-guest-02 ~]$ rpm --verify NetworkManager
missing     /etc/NetworkManager/conf.d
missing     /etc/NetworkManager/dispatcher.d/no-wait.d
missing     /etc/NetworkManager/dispatcher.d/pre-down.d
missing     /etc/NetworkManager/dispatcher.d/pre-up.d
missing     /etc/NetworkManager/dnsmasq-shared.d
missing     /etc/NetworkManager/dnsmasq.d
missing     /etc/NetworkManager/system-connections
missing     /etc/sysconfig/network-scripts
missing     /usr/lib/NetworkManager/VPN
missing     /usr/lib/NetworkManager/conf.d
missing     /usr/lib/NetworkManager/dispatcher.d/no-wait.d
missing     /usr/lib/NetworkManager/dispatcher.d/pre-down.d
missing     /usr/lib/NetworkManager/dispatcher.d/pre-up.d
missing     /usr/lib/NetworkManager/system-connections
```
Any ideas how this can happen?

**Comment by gicmo** [comment]
Possible fix for this might be #871, see the description for details.
